src/utils/index.ts

```diff
@@ -22,7 +22,8 @@ const ETHERSCAN_PREFIXES: { [chainId in ChainId]: string } = {
   3: 'ropsten.',
   4: 'rinkeby.',
   5: 'goerli.',
-  42: 'kovan.'
+  42: 'kovan.',
+  100: 'xdai.'
```

Comment:
Do you need to change it here?
Since it won't be using etherscan for xdai at all?

Reply:
Probably for types and chainID

Reply:
Yes, is only for the types (doesn't compile if I don't add it). 

 Although, theoretically I could add there an empty string, because it's not used. I decided against empty string to keep it more coherent and readable. Also, if Etherescan ever support xDAI the name of the URL will be probably with that prefix.